signer/src/block_observer.rs

```diff
@@ -822,14 +822,14 @@ mod tests {
         // invalid requests (even though it should've validated them) and
         // BitcoinClient will return the right transaction for both of
         // them.
-        let tx_setup0 = sbtc::testing::deposits::tx_setup(lock_time, max_fee, amount);
+        let tx_setup0 = sbtc::testing::deposits::tx_setup(lock_time, max_fee, &[amount]);
         let deposit_request0 = CreateDepositRequest {
             outpoint: bitcoin::OutPoint {
                 txid: tx_setup0.tx.compute_txid(),
                 vout: 0,
             },
-            deposit_script: tx_setup0.deposit.deposit_script(),
-            reclaim_script: tx_setup0.reclaim.reclaim_script(),
+            deposit_script: tx_setup0.deposits.first().unwrap().deposit_script(),
```

Comment:
nit: we should probably add a `first_deposit` and some other util to `TxSetup` to streamline stuff like this -- not in this PR